src/sentry/tsdb/snuba.py

```diff
@@ -77,7 +77,8 @@ def get_data(self, model, keys, start, end, rollup=None, environment_id=None,
         end = to_datetime(series[-1] + rollup)
 
         result = snuba.query(start, end, groupby, None, keys_map,
-                             aggregations, rollup, referrer='tsdb')
+                             aggregations, rollup, referrer='tsdb',
+                             is_grouprelease=(model == TSDBModel.frequent_releases_by_group))
```

Comment:
I really hate this `is_grouprelease` thing, but there's not really a much better way to account for the fact that some callers pass us `Release` and some pass `GroupRelease` but they both use the same `tags[sentry:release]` column